### PR TITLE
Reattempting this pull request.

### PR DESCRIPTION
**Pull request by stevenhuynh17**
Starting from scratch.

### CODE REVIEW

File: client/Components/App.jsx [hunk 1]
```diff
@@ -1,9 +1,35 @@
 import React from 'react';
+<<<<<<< HEAD
```
Comment (turbio) [inline client/Components/App.jsx]: just fyi you're trying to push code that contains an unresolved merge conflict

File: package.json [hunk 2]
```diff
@@ -26,7 +26,12 @@
     "pg": "^6.1.0",
     "react": "^15.3.1",
     "react-dom": "^15.3.1",
+<<<<<<< HEAD
```
Comment (turbio) [inline package.json]: this is why npm install is unable to run

**Comment by jon-is-learning** [comment]
It looks like there are some linter errors. 

You can lint your work before upload by running `npm run build` in the terminal.

**Comment by turbio** [comment]
in your latest commit the stateless components aren't returning anything

**Pull request merged**